KerbalStuff/celery.py

```diff
@@ -84,6 +87,18 @@ def update_from_github(working_directory, branch, restart_command):
         site_logger.exception('Unable to update from github')
 
 
+@app.on_after_configure.connect
+def setup_periodic_tasks(sender, **kwargs):
+    sender.add_periodic_task(600, calculate_mod_scores.s(), name='calculate mod scores')
```

Comment:
There are two score components that depend on time.

- 100 point bonus for creation within last 30 days
- 1 point penalty per 5 days since last update, up to a limit of 100 points

Given this, a 10 minute interval is extreme overkill. We could do 5 days and expect it to work fine.